code/modules/urist/modules/materials/recipes_skin.dm

```diff
@@ -1,3 +1,63 @@
+/material/skin/goat/generate_recipes(reinforce_material)
+	. = ..()
+	if(reinforce_material)	//recipies below don't support composite materials
+		return
```

Comment:
`reinforce_material` check is already carried out in the parent proc, just pass it through
```suggestion
	. = ..(reinforce_material)
```